### ISSUE TITLE
Provide a little margin

### ISSUE THREAD

**Issue by bps** [rogual/neovim-dot-app#175]
To my eyes, it's jarring to see text run directly to the edges of the window. In MacVim.app, there appears to be a two pixel margin on the left, and one pixel on the top, bottom, and right. This seems to give enough breathing room.

Would you be open to adding a little space? If so, point me at the source (`view.mm`?) and I'll try to get a PR in.

**Comment by rogual** [comment]
Good idea, I'd be totally happy to merge this if implemented. Gotta warn you though, our text drawing code is pretty gnarly at the moment.

One thing I'd say is make sure the margin matches the background colour of the first cell on that line.

Text drawing code is here: https://github.com/rogual/neovim-dot-app/blob/master/src/redraw.mm#L77

**Comment by philplckthun** [comment]
@rogual @bps I don't see an efficient solution to this atm as we're resourcing the nvimrc and thus the colorscheme currently, to circumvent the gui_running problem. Thus we probably have to listen for an event and then somehow get the right background color. Maybe the colorscheme event in combination with sending something via msgpack could work? Then we could get the background color from the appropriate vim variable instead of the first cell.
